File: covid/Model/People/Person.cs
using Model.Generics;
using Model.MasterModel;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Model.People
{
    public class Person : Generic
    {
        public string IdCountry { get; set; }
        public string IdResult { get; set; }
        public string IdDna { get; set; }
        public Country Country { get; set; }
        public Result Result { get; set; }
        public List<Dna> Dna { get; set; }

    }
}

File: CovidFrontApp/src/app/core/person.ts
import { Generic } from "./generic";
export class Person extends Generic{
  country:string;
  dna:Array<string>;
  result:string;

  constructor(item:any){
    super(item.id, item.name);
    this.country = item.country;
    this.dna = item.dnas;
    this.result = item.result;
  }
}
